BT-Tracking/Interface/Active/ActiveAppVC.swift

```diff
@@ -149,30 +160,40 @@ final class ActiveAppVC: UIViewController {
     
     @objc private func applicationDidBecomeActive() {
         checkForBluetooth()
+        checkForExposureService()
+        if viewModel.state == .disabledExposures {
+            updateScanner()
+        }
     }
 }
 
 private extension ActiveAppVC {
 
-    func updateViewModel() {
+    func updateViewModel(updateScanner: Bool = true) {
         viewModel = ActiveAppVM(bluetoothActive: viewModel.lastBluetoothState)
 
-        updateScanner()
+        if updateScanner {
+            self.updateScanner()
+        }
         updateInterface()
         setupStrings()
     }
 
     func updateScanner() {
         switch viewModel.state {
-        case .enabled, .disabled:
+        case .enabled, .disabledBluetooth, .disabledExposures:
             viewModel.exposureService.activate { [weak self] error in
+                self?.checkForExposureService()
+
                 guard let error = error else { return }
-                self?.show(error: error)
+                log("ActiveAppVC: failed to active exposures \(error)")
             }
         case .paused:
             viewModel.exposureService.deactivate { [weak self] error in
```

Comment:
Musi se na puse cely exposure deactivovat? Nestaci ho dat do stavu paused?

Reply:
Pause je interni, to je jedina metoda ktera je open